### ISSUE TITLE
Web - Search page - First item is highlighted once any value is entered/deleted in the field

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#9604]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Issue was found when executing PR** https://github.com/Expensify/App/pull/7702

## Action Performed:
1. Login to https://staging.new.expensify.com/
2. Open search page/new group/new chat/split bill
3. Highlight any result via arrow buttons
4. Start typing any text in the search fields and delete it

## Expected Result:
Once text in search field is deleted – the focus should be where you left it and you should be able to use the arrows to navigate focus (as before). If highlighted item still present in search results - it's should be stay focused
The same for any page with user searching: new chat, new group, split bill, search page, workspace invitation

## Actual Result:
First item is highlighted once any value is entered in Search field

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.79.2

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** any

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/176259609-06e0d2f8-2d7b-47ea-a268-d242b5ef2de5.mp4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause - Internal Team
 
**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by iwiznia** [comment]
I don't think this is an issue, saving the position of your selected item after the search has changed is not needed and I don't think it would even be expected. IMO the current behavior is correct.
